lib/src/plugin/heat_map_tiles_provider.dart

```diff
@@ -63,8 +65,10 @@ class HeatMapTilesProvider extends TileProvider {
 
     var localMin = 0.0;
     var localMax = 0.0;
-    CustomPoint tileOffset =
-        CustomPoint(options.tileSize * coords.x, options.tileSize * coords.y);
+    // CustomPoint tileOffset =
+    // CustomPoint(options.tileSize * coords.x, options.tileSize * coords.y);
```

Comment:
I would suggest removing the old code entirely so it doesn't clog up the codebase as it won't be used again

```suggestion
```